export_job/export_zip_from_manifest_job.py

```diff
@@ -129,6 +123,9 @@ def fail(error_message=DEFAULT_ERROR_MESSAGE):
 if __name__ == "__main__":
     access_token = os.environ["ACCESS_TOKEN"]
     hostname = os.environ["GEN3_HOSTNAME"]
+    bucket_name = os.environ["BUCKET"]
```

Comment:
maybe consider failing the job if this env var is not set?

Reply:
That is a great idea! I just updated the code to reflect this!